### DISCUSSION TITLE
do newer versions of Python and Pytorch work for Pytorch3D ?

### DISCUSSION THREAD

**Discussion by shayansep** [facebookresearch/pytorch3d#1806] [General]
Now, can we use Python version 3.12.2 and Pytorch 2.2.2+cu121 ?? or we should only use the exact versions that was mentioned on Github ?

**Comment by bottler** [comment]
The `main` branch of pytorch3d will almost always work fine with more recent versions of python and pytorch than we advertise. This is because internally pytorch3d and pytorch are used in a monorepo together so they develop together. But you will have to build from source. There usually won't be built packages on anaconda for newer versions than advertised - and if there are, that would probably be a documentation bug!
